src/views/RealTimeEvents/Blocks/azion-query-language.vue

```diff
@@ -23,37 +23,17 @@
             size="small"
             class="h-auto w-full md:max-w-fit"
             @click="executeQuery"
-            :disabled="formattedQuery.length !== 0"
+            :disabled="handleErrrosQuery.length !== 0"
```

Comment:
Typo. Update where necessary
```suggestion
            :disabled="handleErrorsQuery.length !== 0"
```